test/integration/builder/inserts.js

```diff
@@ -702,6 +702,15 @@ module.exports = function(knex) {
         })
       });
 
+      it('transaction.batchInsert using specified transaction', function() {
+        return knex.transaction(function(tr) {
+          tr.batchInsert('BatchInsert', items, 30)
+          .returning(['Col1', 'Col2'])
+          .then(tr.commit)
+          .catch(tr.rollback);
```

Comment:
@jurko-gospodnetic Like you mentioned, I just copied/pasted an existing test.  I'll leave what happens to the tests up to the maintainers.